contributor-guides/	translating-astro-docs.md

```diff
@@ -0,0 +1,359 @@
+
+### How can I find out what needs to be reviewed/translated?
+
+To find translations pending review, you can filter through this repo's [Pull Requests with the i18n label](https://github.com/withastro/docs/pulls?q=is%3Apr+is%3Aopen+label%3Ai18n).
+
+See our automated [Translation Status Overview](https://i18n.docs.astro.build/) for a quick list of which pages are missing a translation or need updating to match a change to the English version.
+
+> **Warning**
+> Please do not translate any pages without first checking their status in our [Overview Issue](https://github.com/withastro/docs/issues/438)! If a page is not listed here as needing a translation or an update, we can not accept your PR.
+ 
+Not every page is marked as "ready to translate." So, even if you find a page that is not yet translated on the Docs site, you must first confirm that it is on the list of available pages to translate. Do not translate documents that are missing:
+
+- A "Translate this page" button in the Docs site.
+- The `i18nReady: true` frontmatter property in its MDX file.
+
+You can read more about how pages are marked "ready for (initial) translating" and "needs updating" in [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md).
+
+## Languages for translation
+
+Currently we are aiming to translate the Astro documentation into the following languages:
+
+- Brazilian Portuguese
+- Chinese (Simplified)
+- English
+- French
+- German
+- Japanese
+- Spanish
+- Korean
+- Polish
+
+### Why can Astro only support a few languages?
+
+Supporting a language means that Astro takes responsibility for maintaining that language going forward. It’s not just a one-time translation, and it is not just a few pages. It is continuous and ongoing updates for both content and site design, maintaining complete versions of the docs in every supported language.
+
+### How do you choose which languages to support?
+
+1. Where our traffic comes from, and the user base we need to support
+2. Availability of community members who can commit time to maintaining the translations
+
+### What about unsupported languages?
+
+The official docs will only contain supported languages for now, but we will be looking to add more officially supported languages as we grow.  If you would like to host translated docs yourself in another language, please let us know! We are happy to help you set this up and coordinate with future changes to the docs.
```

Comment:
```suggestion
The official docs will only contain supported languages for now, but we will be looking to add more officially supported languages as we grow. If you would like to host translated docs yourself in another language, please let us know! We are happy to help you set this up and coordinate with future changes to the docs.
```

Reply:
I don't understand the change that you made here @dreyfus92

Reply:
Just deleted an extra space that you had before the `if you would...`

Reply:
> Just deleted an extra space that you had before the `if you would...`

Ahh ok. I had that feeling, but I just wanted to be sure.